docs/UserGuide.adoc

```diff
@@ -513,20 +513,20 @@ __Constraints:__
 // tag::list[]
 ==== Listing tasks : `list` +
 
-Want to go back to the default list of task after a search? Simply use the list command:
+When you want to go back to the default list of tasks after a search, you can use the list command to return to the default list of tasks.
```

Comment:
the "go back to the default list of tasks" phrase has been repeated twice. You can just put a full stop after list command.

Reply:
I think the original statement was good, so if you want to keep that you can just replace the colon with a full stop.